Mathlib/Topology/Homeomorph.lean

```diff
@@ -49,139 +49,140 @@ infixl:25 " ≃ₜ " => Homeomorph
 
 namespace Homeomorph
 
-variable [TopologicalSpace α] [TopologicalSpace β] [TopologicalSpace γ] [TopologicalSpace δ]
+variable [TopologicalSpace X] [TopologicalSpace Y] [TopologicalSpace Z]
+  {X' Y' : Type*} [TopologicalSpace X'] [TopologicalSpace Y']
 
-theorem toEquiv_injective : Function.Injective (toEquiv : α ≃ₜ β → α ≃ β)
+theorem toEquiv_injective : Function.Injective (toEquiv : X ≃ₜ Y → X ≃ Y)
   | ⟨_, _, _⟩, ⟨_, _, _⟩, rfl => rfl
 #align homeomorph.to_equiv_injective Homeomorph.toEquiv_injective
 
-instance : EquivLike (α ≃ₜ β) α β where
+instance : EquivLike (X ≃ₜ Y) X Y where
   coe := fun h => h.toEquiv
   inv := fun h => h.toEquiv.symm
   left_inv := fun h => h.left_inv
   right_inv := fun h => h.right_inv
   coe_injective' := fun _ _ H _ => toEquiv_injective <| FunLike.ext' H
 
-instance : CoeFun (α ≃ₜ β) fun _ ↦ α → β := ⟨FunLike.coe⟩
+instance : CoeFun (X ≃ₜ Y) fun _ ↦ X → Y := ⟨FunLike.coe⟩
 
 @[simp]
-theorem homeomorph_mk_coe (a : Equiv α β) (b c) : (Homeomorph.mk a b c : α → β) = a :=
+theorem homeomorph_mk_coe (a : Equiv X Y) (b c) : (Homeomorph.mk a b c : X → Y) = a :=
```

Comment:
Why was notation not used here for `Equiv`? Fixing it is in the scope of this PR, I think.

Reply:
Done. Also renamed `a` to `h`.